src/main/java/au/org/ala/biocache/dto/OccurrenceIndex.java

```diff
@@ -92,17 +92,17 @@
     @Field("institutionUid")
     String institutionUid;
     @Field("institutionCode")
-    String raw_institutionCode;
+    String institutionCode;
     @Field("institutionName")
     String institutionName;
     @Field("collectionUid")
     String collectionUid;
     @Field("collectionCode")
-    String raw_collectionCode;
+    String collectionCode;
     @Field("collectionName")
     String collectionName;
     @Field("catalogNumber")
-    String raw_catalogNumber;
+    String catalogNumber;
```

Comment:
I think this effectively changes the API (unless Im missing something) as it'll change the JSON in the basic search service. 

e.g. https://biocache.ala.org.au/ws/occurrences/search?pageSize=1